app/assets/javascripts/discourse/app/mixins/composer-upload-uppy.js

```diff
@@ -634,17 +634,19 @@ export default Mixin.create(ExtendableUploader, UppyS3Multipart, {
       this.mobileUploadButton = document.getElementById(
         this.mobileFileUploaderId
       );
-      this.mobileUploadButtonEventListener = () => {
-        document.getElementById(this.fileUploadElementId).click();
-      };
-      this.mobileUploadButton.addEventListener(
+      this.mobileUploadButton?.addEventListener(
         "click",
         this.mobileUploadButtonEventListener,
         false
       );
     }
   },
 
+  @bind
+  _mobileUploadButtonEventListener() {
```

Comment:
Does it matter that this is moving from `mobileUploadButtonEventListener` to `_mobileUploadButtonEventListener` but line 639 still references `mobileUploadButtonEventListener`?